content/metadata.yaml

```diff
@@ -345,10 +345,15 @@ authors:
   orcid: 0000-0003-3056-4360
   twitter: APoetsch
   email: arpoetsch@gmail.com
-  contributions: Formal Analysis
+  contributions:
+  - Formal Analysis
+  - Funding acquisition
```

Comment:
Just saw this PR come in and wondered if we should add `Writing - Review and editing` as a contribution here, and similarly for everyone who sends back edits to incorporate.